app/utils/getCacheAnalysis.test.ts

```diff
@@ -66,6 +66,39 @@ describe('getCacheAnalysis', () => {
 
     expect(() => getCacheAnalysis(headers, now)).toThrow('Could not determine who served the request')
   })
+
+  it('handles cache miss without function headers (issue reproduction)', () => {
+    // This reproduces the exact headers from the issue where no cache analysis is rendered
```

Comment:
don't think we need a regression test like this. let's describe and test the missing behaviour.